### PR TITLE
envknob, ipn/ipnlocal: add SSH admin kill switch

### PR DESCRIPTION
**Pull request by bradfitz**
Updates #3802

### CODE REVIEW

File: envknob/envknob.go [hunk 1]
```diff
@@ -142,3 +142,6 @@ func LookupInt(envVar string) (v int, ok bool) {
 // UseWIPCode is whether TAILSCALE_USE_WIP_CODE is set to permit use
 // of Work-In-Progress code.
 func UseWIPCode() bool { return Bool("TAILSCALE_USE_WIP_CODE") }
+
+// CanSSHD is whether the Tailscale SSH server is allowed to run.
```
Comment (maisem) [inline envknob/envknob.go:146]: Can you clarify what happens if ssh is already enabled?

Comment (bradfitz) [reply envknob/envknob.go:146]: It doesn't start.

Comment (bradfitz) [reply envknob/envknob.go:146]: It's not allowed to run. I think the comment's pretty clear already.

Comment (bradfitz) [reply envknob/envknob.go:146]: That is, it says nothing about state transitions here at all.

Comment (bradfitz) [reply envknob/envknob.go:146]: PTAL

**Review by maisem — APPROVED** [review]

**Pull request merged**